clients/extjs/js/stigmanUtils.js

```diff
@@ -1691,7 +1691,7 @@ function uploadStigs(n) {
 		   
 						let contents = await parentZip.loadAsync(f)
 						let fns = Object.keys(contents.files)
-						let xmlMembers = fns.filter( fn => fn.toLowerCase().endsWith('xccdf.xml') || fn.toLowerCase().endsWith('benchmark.xml') )
+						let xmlMembers = fns.filter( fn => fn.toLowerCase().endsWith('.xml'))
```

Comment:
As you said, this ends up throwing all XML members to the API and makes the API sort things out. Instead of just throwing everything over the wall, it might be more efficient (for the API at least) and a better user experience (in the client) to implement basic vetting of the file in the client. Only send the API something we reasonably think is valid.

Maybe we can iterate to that. I haven't run this client code yet, maybe this is acceptable for now.

Reply:
I think I'll leave this as something we will iterate too if needed